src/matter/cluster/server/OperationalCredentialsServer.ts

```diff
@@ -107,14 +107,30 @@ export const OperationalCredentialsClusterHandler: (conf: OperationalCredentials
             fabricIndex: fabric.fabricIndex,
         })));
 
+        // TODO persist fabrics
+
         return {status: OperationalCertStatus.Success};
     },
 
-    removeFabric: async ({ request: {fabricIndex}, session }) => {
+    removeFabric: async ({ request: {fabricIndex}, attributes: {fabrics}, session }) => {
         const device = session.getContext();
-        device.removeFabric(fabricIndex);
-
-        return {status: OperationalCertStatus.Success};
+        if (device.removeFabric(fabricIndex)) {
+            fabrics.setLocal(device.getFabrics().map(fabric => ({
```

Comment:
Now that this is supported, I might probably be better to make "fabrics" attribute a getterAttribute (value generated on the fly, like "currentFabric", I just added support), rather than an AttributeServer holding a value.
This would avoid to duplicate setting this attribute everywhere to match the current list of fabrics.

Reply:
Done. pairing with iOS still works